CardScan/Classes/BundleURL.swift

```diff
@@ -1,10 +1,17 @@
-import CoreML
-
-@available(iOS 11.2, *)
-struct BundleURL{
-    static func compiledModel(forResource: String, withExtension: String) -> URL? {
+public struct BundleURL {
+    // If you change the bundle name make sure to set these before
+    // initializing the library
+    public static var bundleName = "CardScan"
+    public static var extensionName = "bundle"
+    public static var cardScanBundle: Bundle?
```

Comment:
Are you setting these values statically to have default values? So it'll always look for CardScan.bundle if not changed. Why not it be empty so it can be filled it like ```BundleUrl(bundleName:"Whatever the user puts in", extenstionName: "whatever")```

Reply:
The goal is to let the app that embeds our SDK specify the bundle name or the actual bundle itself to give them more flexibility in their build system. Thus, BundleURL shouldn't even get called externally (except for testing), it's only internal so we're a little constrained, although I do agree with your sentiment.